addons/point_of_sale/report/pos_invoice.py

```diff
@@ -23,4 +23,7 @@ def get_report_values(self, docids, data=None):
             not_invoiced_orders_names = [a.name for a in not_invoiced_posorders]
             raise UserError(_('No link to an invoice for %s.') % ', '.join(not_invoiced_orders_names))
 
-        return self.env['ir.actions.report'].sudo().render_template('account.report_invoice', {'docs': self.env['account.invoice'].sudo().browse(ids_to_print)})
+        return {
+            'docs': self.env['account.invoice'].sudo().browse(ids_to_print),
+            'template_name': 'account.report_invoice'
+        }
```

Comment:
Hello @smetl , 

If i will do Change as your suggestion then when I am printing invoice report from account then it will call this method also and givinng me error like : https://drive.google.com/file/d/0B4ek8t9nTzXiX0J6cTBCTFhyOWs/view?usp=drivesdk Thanks